src/scripts/autocast.js

```diff
@@ -26,7 +26,7 @@ export function isEnabled() {
     const playerId = localStorage.getItem('autocastPlayerId');
     const currentPlayerInfo = playbackManager.getPlayerInfo();
 
-    return (currentPlayerInfo && playerId && currentPlayerInfo.id === playerId);
+    return currentPlayerInfo?.id === playerId;
```

Comment:
This would actually behave differently in the case that both `currentPlayerInfo` and `playerId` are undefined... I think we need to leave one of the checks to prevent that...
```suggestion
    return playerId && currentPlayerInfo?.id === playerId;
```